app/src/main/res/drawable/search_widget_favorite_favicon_light_background.xml

```diff
@@ -15,6 +15,6 @@
   -->
 
 <shape xmlns:android="http://schemas.android.com/apk/res/android">
-    <solid android:color="@color/whiteFive" />
+    <solid android:color="@color/black6" />
     <corners android:radius="10dp" />
```

Comment:
is this 10dp? can we use an already set radius?

Reply:
ditto